examples/tv-casting-app/linux/main.cpp

```diff
@@ -252,11 +252,11 @@ void DeviceEventCallback(const DeviceLayer::ChipDeviceEvent * event, intptr_t ar
             ChipLogError(AppServer, "Associate() failed: %" CHIP_ERROR_FORMAT, err.Format());
             return;
         }
-        LaunchURL::Type request;
-        request.contentURL          = chip::CharSpan(kContentUrl, strlen(kContentUrl));
-        request.displayString       = chip::CharSpan(kContentDisplayStr, strlen(kContentDisplayStr));
-        request.brandingInformation = chip::app::DataModel::List<
-            const chip::app::Clusters::ContentLauncher::Structs::ContentLaunchBrandingInformation::Type>();
+        LaunchURLRequest::Type request;
+        request.contentURL    = chip::CharSpan(kContentUrl, strlen(kContentUrl));
+        request.displayString = chip::CharSpan(kContentDisplayStr, strlen(kContentDisplayStr));
+        request.brandingInformation =
+            chip::app::DataModel::List<const chip::app::Clusters::ContentLauncher::Structs::BrandingInformation::Type>();
```

Comment:
Fwiw, empty list is the default anyway, so this line is probably unnecessary.

Reply:
I'm not the author, I'll leave it as it might be useful to the author.